src/video/sdl2_v.cpp

```diff
@@ -243,6 +243,18 @@ void VideoDriver_SDL_Base::EditBoxLostFocus()
 	}
 }
 
+std::vector<int> VideoDriver_SDL_Base::GetListOfMonitorRefreshRates()
+{
+	std::vector<int> rates = {};
+	int displayCount = SDL_GetNumVideoDisplays();
+	for (int i = 0; i < displayCount; i++) {
```

Comment:
as `dispalycount` is only used once, you can put the function call in the `if`. That also makes it a bit more like `FindResolutions`.

Talk about that, you check the error code, that function doesn't for `SDL_GetDisplayMode` .. not something for you to fix perse, but interesting nevertheless.